tests/unit/test_charm.py

```diff
@@ -308,3 +308,37 @@ def test_pebble_ready(harness: Harness, patch: SpringBootPatch):
     harness.begin_with_initial_hooks()
     harness.container_pebble_ready("spring-boot-app")
     assert isinstance(harness.model.unit.status, ActiveStatus)
+
+
+def test_nginx_ingress(harness: Harness, patch: SpringBootPatch):
```

Comment:
Rename to `test_ingress` only I think to make it generic as the same test will work for traefik

Reply:
Updated, thanks!